src/style.css

```diff
@@ -0,0 +1,40 @@
+body {
+  background: black;
+  margin: 0;
+}
+
+.en-lang {
+  width: 328px;
+}
+
+.ua-lang {
+  width: 328px;
+}
+
+.ru-lang {
+  width: 328px;
+}
+
+.blue-border {
+  padding: 10px;
+  margin: 20px 20px 40px;
+  background: rgb(48, 141, 192);
+  border: 1px solid rgb(231, 85, 81);
+}
+
+.yellow-border {
+  background: rgb(255, 165, 0);
+  padding: 10px;
+  border: 1px solid rgb(231, 85, 81);
+}
```

Comment:
The theme of the site may be changed and binding class names to colors is bad practice. You can think of these blocks as content, wrapper, and inner wrapper. Example: outer, inner, wrapper, content, etc.

Reply:
okey